docs/docs/cmd/planner/bucket/bucket-set.md

```diff
@@ -17,16 +17,16 @@ m365 planner bucket set [options]
 : Name of the bucket. Specify either `id` or `name` but not both.
 
 `--planId [planId]`
-: ID of the plan to update the bucket of. Use when referring to bucket using `name`. Specify either `planId` or `planName` but not both.
+: ID of the plan to update the bucket of. Specify either `planId` or `planName` when using `name`.
 
 `--planName [planName]`
-: Name of the plan to update the bucket of. Use when referring to bucket using `name`. Specify either `planId` or `planName` but not both.
+: Name of the plan to update the bucket of. Specify either `planId` or `planName` when using `name`.
```

Comment:
Let's add 'Always use in combination with either `ownerGroupId` or `ownerGroupName`.` to make it more clear what you need when you specify this option, and to align it with issue #3347.

Reply:
Should we maybe implement this in a new issue. Because there are a lot of commands where this suggestion could apply.

Reply:
For now let's just change this one as we're currently working on it. We can create a new issue for other commands if necessary.